src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageQueryParams.cs

```diff
@@ -14,5 +14,7 @@ internal class RazorLanguageQueryParams : IRequest<RazorLanguageQueryResponse>
         public Uri Uri { get; set; }
 
         public Position Position { get; set; }
+
+        public bool FindNextCSharpPositionForHtml { get; set; }
```

Comment:
Naming suggestions most welcome 😛

I originally thought a new LSP method would be appropriate, but it would have been a lot of copied code that was no sharable, so this is _way_ simpler.